src/mon/Elector.cc

```diff
@@ -151,6 +151,8 @@ void Elector::reset_timer(double plus)
    */
   expire_event = mon->timer.add_event_after(
     g_conf()->mon_election_timeout + plus,
+    // TODO: uh, this means when we _defer_to() above that we give them a whole
+    // new election period; is that intentional?
```

Comment:
Oh @liewegas I meant to ask you about this verbally. Any thoughts?

Reply:
The line
```
  reset_timer(1.0);  // give the leader some extra time to declare victory
```
dates to 2007, with no explanation given.  I don't really understand why the leader would need more than a full interval.  At the time is was ``mon_lease`` and not ``mon_election_timeout``... maybe that's why?  Anyway, this can  probably be removed.